Cargo.lock

```diff
@@ -278,15 +271,14 @@ dependencies = [
  "jni",
  "log",
  "once_cell",
- "paste",
  "rustls",
  "rustls-native-certs",
  "rustls-platform-verifier-android",
  "rustls-webpki",
  "security-framework",
  "security-framework-sys",
  "webpki-root-certs",
- "windows-sys 0.52.0",
+ "windows-sys 0.59.0",
```

Comment:
Suggestion: I believe this line needs reverted or otherwise we won't be getting CI coverage over both 0.52 and 0.59 to match the semver range specified in the manifest.

Reply:
Ah, good catch. Reverted.